### PR TITLE
Warhacks adaptation

### PR DESCRIPTION
**Pull request by qiqi0725**
Landing page with content and minor styling.
Waiting for logo to finalize styling

### CODE REVIEW

File: app/client/views/home/home.html [hunk 1]
```diff
@@ -155,11 +199,16 @@ <h1>faq</h1>
 				<div class="page-header">
 					<h1>contact</h1>
 				</div>
-				<div>
-					"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-					Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute
-					irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
-					cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
+				<div class="page-text">
+					<ul class="contact-list">
```
Comment (alvyn279) [inline app/client/views/home/home.html]: @qiqi0725 
I tried this on my local. Does this look better as a contact display list?

image

If yes then I'll push it

File: app/client/views/home/home.html [hunk 2]
```diff
@@ -128,23 +110,85 @@ <h1>sponsors</h1>
...
-					cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
+				<div class="page-text text-center row">
+					<div class="col-lg-4 col-md-4 col-sm-6 col-xs-12">
+						<div class="question">
+							What is a hackathon?
+						</div>
+						<div class="answer">
+							Although many people associate the word hacking with illegal activities a hackathon is completely the opposite. A hackathon
+							is event where people can come together and share their ideas to create something fun while also learning and having
+							a great time.
```
Comment (alvyn279) [inline app/client/views/home/home.html]: Typos:
"Although many people associate the word hacking with illegal activities **,**[HERE] a hackathon is completely the opposite. A hackathon is **an** [HERE] event where.."

File: app/client/views/home/home.html [hunk 3]
```diff
@@ -128,23 +110,85 @@ <h1>sponsors</h1>
...
+							other participants and form a team with cool new people.
+						</div>
+					</div>
+					<div class="col-lg-4 col-md-4 col-sm-6 col-xs-12">
+						<div class="question">
+							What should I bring?
+						</div>
+						<div class="answer">
+							We don’t ask you to bring anything because we will be providing you with all the equipment you need. However, it is suggested
+							that if you intend to program you bring your own laptop already set up with the Arduino IDE so that you can immediately
```
Comment (alvyn279) [inline app/client/views/home/home.html]: Typo:
"suggested that if you intend to program, [HERE] you "

File: app/client/views/home/home.html [hunk 4]
```diff
@@ -128,23 +110,85 @@ <h1>sponsors</h1>
...
+							start coding during the competition. You are also welcome to bring any hardware you want to experiment with at the
+							hackathon as long as it’s safe, contact us if you are unsure.
```
Comment (alvyn279) [inline app/client/views/home/home.html]: Typo:
"... it’s safe; [HERE]contact us if you are unsure"

File: app/client/views/home/home.html [hunk 5]
```diff
@@ -128,23 +110,85 @@ <h1>sponsors</h1>
...
+							John Molson Building at the Sir George Williams Campus.
+						</div>
+					</div>
+					<div class="col-lg-4 col-md-4 col-sm-6 col-xs-12">
+						<div class="question">
+							Can I participate?
+						</div>
+						<div class="answer">
+							If you are a college student or a high school senior, then you are eligible to attend! No experience is necessary to participate
+							in this hackathon. Just remember if you are a minor on the day of the competition you will have get consent from a
```
Comment (alvyn279) [inline app/client/views/home/home.html]: Typo:
"Just remember, [HERE] if you are a minor on the day"

File: app/client/views/home/home.html (hunk 1 above)
Comment (qiqi0725) [reply app/client/views/home/home.html]: Looks good. Can you add email as well

Comment (alvyn279) [reply app/client/views/home/home.html]: Yes will do

**Pull request merged**